assignment.md

```diff
@@ -11,47 +11,62 @@ If the version control concepts or `git` commands are still confusing to you, co
  0. Create a branch in this assignment repository named `submission`, then list all the branches you have (Hint: look at the Branch chapter of notes on how branches work, how to create new ones, and check existing branches):
 
     ```bash
-    # git command used and new list of branches
+    # git command used: `git checkout -b submissions
+    # list of branches: 
+	main
+	*submission
     ```
 
  1. List the current `remote` name and hyperlink destination of this assignment repository (Hint: if you are not sure, add `--help` after your command to check remotes.). 
 
     ```bash
-    # your git command and output here
+    # git command: git remote get-url origin
+    # output: https://github.com/asimon3210/version_control_assignment.git   
     ```
 
 
  2. Add a second remote destination named `upstream` to your assignment repository, `upstream` is a common name used for the repository that people have forked from, in our case, the `upstream` will be at `https://github.com/WCM-datascibasics/version_control_assignment`. List the new list of remotes after you've added the new remote. 
  
     ```bash
-    # list the new list of remotes here
+    # origin  https://github.com/asimon3210/version_control_assignment.git (fetch)                                            origin  https://github.com/asimon3210/version_control_assignment.git (push)                                             upstream        https://github.com/WCM-datascibasics/version_control_assignment (fetch)                                 upstream        https://github.com/WCM-datascibasics/version_control_assignment (push) 
     ```
 
  3. Save your modifications to this file so far and create a commit indicating you've answered the first 2 questions. Then try pushing the changes to the `upstream` destination on GitHub. What happens? Explain in your own words why does this happen? What are the benefits of having this `upstream` remote added when working collaboratively (Hint: read git command outputs!)?
 
-    > Your answer here
-
+    > I get the following error:
+	remote: Permission to WCM-datascibasics/version_control_assignment.git denied to asimon3210.
+	fatal: unable to access 'https://github.com/WCM-datascibasics/version_control_assignment/': The requested URL returned error: 403
+        This is probably because I used an incorrect password to access the WCM-datascibasics account. I didn't have the correct credentials, so I did not have permission to access this remote. 
```

Comment:
It's not the password, its simply ownership, you own `asimon3210`'s repositories, but not `wcm-datascibasics`'s respositories. But also, you didn't answer what are the benefits of having an `upstream` remote?

If `wcm-datascibasics` made an update to the assignment repository's `main` branch, you can pull directly from `wcm-datascibasics` by doing `git pull upstream main`, instead of having to re-fork the repository to include my updates.  

-5%.

Reply:
OH you made the pull request for `wcm-datascibasics/main` <- `asimon3210/submission`, I wanted a merge for your own repository, meaning `asimon3210/main`<- `asimon3210/submission`.